README.md

```diff
@@ -25,3 +25,12 @@ Contains im/exporters to convert models between LionCore &harr; Ecore and LionWe
 Some of LionCore's built-in elements have no direct representation in Ecore.
 This sub-project is an Eclipse project that defines an _EPackage_ `builtins` to host the equivalent elements in Ecore.
 The language's _nsURI_ is `http://lionweb.io/lionweb-java/emf/core/builtins/2023.1`.
+
+## Changelog
+
+## lionweb-java-2023.1-* - Version 0.2.0
+
+This is the first release based on the new coordinates, which indicate support for the specification of LionWeb released under version 2023.1.
```

Comment:
“coordinate**s**”? I'm not all too familiar with Maven parlance, so I could miss something. Could “package name” or something like that be more clear?

Reply:
The term coordinates is indeed part of the Maven parlance (https://maven.apache.org/pom.html#Maven_Coordinates). It includes the groupId, the artifact id, and the version. I can change that to artifactIds, because we changed only those